src/tuntap/windows/c_tuntap_windows.cpp

```diff
@@ -48,6 +49,44 @@ void c_tuntap_windows_obj::async_receive_from_tun(unsigned char *const data, siz
 void c_tuntap_windows_obj::set_tun_parameters(const std::array<unsigned char, 16> &binary_address, int prefix_len, uint32_t mtu) {
 	_fact("Setting IPv6 address, prefixLen=" << prefix_len);
 	std::wstring human_name = get_human_name(m_guid);
+	NET_LUID luid = get_luid(human_name);
+	_fact("Setting address on human_name " << to_string(human_name));// << " luid=" << to_string(luid));
+																	 // remove old address
+	MIB_UNICASTIPADDRESS_TABLE *table = nullptr;
+	NETIOAPI_API status = GetUnicastIpAddressTable(AF_INET6, &table);
+	_check_sys(status == NO_ERROR);
+	for (int i = 0; i < static_cast<int>(table->NumEntries); ++i) {
```

Comment:
statis_assert(  ...limits..<ULONG>::max() <=  .....<size_t>::max() )  // ULONG - https://msdn.microsoft.com/en-us/library/windows/desktop/aa814508%28v=vs.85%29.aspx
for (   size_t i = ...                                       static_cast<size_t>  ...